.github/workflows/snyk_scan.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: vanagon check PRs
+
+on: [pull_request]
+
+jobs:
+ snyk_vanagon:
+   runs-on: ubuntu-latest
+   steps:
+    - name: checkout the current PR
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        persist-credentials: false
+    - name: Run Vanagon Snyk Scan
+      uses: puppetlabs/security-snyk-vanagon-action@v2.2.0
+      with:
+        snykToken: ${{ secrets.SNYK_PE_TOKEN }}
+        snykOrg: 'puppet-enterprise'
+        skipPlatforms: 'debian-10-armhf'
+        rproxyUser: ${{ secrets.RPROXY_USER }}
```

Comment:
I realized that these secrets aren't getting interpolated properly because there's a discrepancy between what's in the code (`RPROXY_USER`) and what those secrets are called in the repo (`SEC_PUBLIC_RPROXY_USER`).

I've changed that in my PR (https://github.com/puppetlabs/pxp-agent-vanagon/pull/43) but am still running into issues, though maybe this will help with your debugging